ci/azure-functions-node.js-linux.yml

```diff
@@ -0,0 +1,40 @@
+name: Azure Functions with NodeJS
+
+on:
+  push:
+    branches: [ $default-branch ]
+  
+  workflow_dispatch:
+
+env:
+  AZURE_FUNCTIONAPP_PACKAGE_PATH: '.'   # set this to the path to your web app project, defaults to the repository root
+  NODE_VERSION: '14.x'                  # set this to the node version to use (supports 8.x, 10.x, 12.x)
```

Comment:
The comment does not match the default value used. Is that restriction a valid one?